front-end-babylonjs/public/message.json

```diff
@@ -0,0 +1,52 @@
+{
+    "Message" : {
+        "ZIG_ZAG" : {
+            "Info" : "Cold-calling: You respond to someone contacting you out of the blue to move their pension" ,
+            "Icon" : "a phone",
+            "reduction" : 400
+        },
+        "SPLITTER" : {
+            "Info" : "Offered withdrawal: Transfer your pension to a scheme because they offer you early access to your pension" ,
+            "Icon" : "a non-standard coloured coin",
+            "reduction" : 800
+        },
+        "ACCELERATOR" : {
+            "Info" : "Accelerator: You engaged with a company because they offered you free financial advice",
+            "Icon" : "pound signs with a cross through it",
+            "reduction" : 300
+        },
+        "BLACK_OUT" : {
+            "Info" : "Black out: You engaged with a company because they offered a unique or limited investment opportunity",
+            "Icon" : "something unusual",
+            "reduction" : 500
+        },
+        "SPEEDY" : {
+            "Info" : "Speedy: You agree to something because you are told the offer has a time limit on it",
+            "Icon" : " clock",
+            "reduction" : 600
+        },
+        "DIAGONAL" : {
+            "Info" : "Diagonal: Fake scammer vigilantes working on people’s fear of scams to get them to move their pension",
+            "Icon" : "mini scam man",
+            "reduction" : 700
+        },
+        "NORMAL_SCAM" : {
```

Comment:
I'm confused about which scam is the basic scam, as we refer to cold call in the game wireframe as the basic scam

Reply:
Normal scam is the one very basic (No special movement) with just a fall.
Cold calling as been identified with ZIG ZAG movement. 

Yes. We understand that normal scam is no more essential. Stage 1 will begin with Cold Calling.(Current version has Stage 0 with Normal Scam that will be removed as per Lucy's flow chart)